packages/fxa-admin-panel/src/components/AccountSearch/index.test.tsx

```diff
@@ -139,18 +139,155 @@ function exampleBounceMutationResponse(email: string): MockedResponse {
   };
 }
 
+const MinimalAccountResponse = (testEmail: string) => ({
+  data: {
+    accountByEmail: {
+      uid: '123',
+      createdAt: 1658534643990,
+      disabledAt: null,
+      emails: [
+        {
+          email: testEmail,
+          isVerified: true,
+          isPrimary: true,
+          createdAt: 1658534643990,
+        },
+      ],
+      emailBounces: [],
+      securityEvents: [],
+      totp: [],
+      recoveryKeys: [],
+      linkedAccounts: [],
+      attachedClients: [],
+      subscriptions: [],
+    },
+  },
+});
+
 beforeEach(() => {
   jest.spyOn(window, 'confirm').mockImplementation(() => true);
   testEmail = chance.email();
 });
 
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
 it('renders without imploding', () => {
   const renderResult = render(<AccountSearch />);
   const getByTestId = renderResult.getByTestId;
 
   expect(getByTestId('search-form')).toBeInTheDocument();
 });
 
+it('calls account search', async () => {
+  let calledAccountSearch = false;
+  const mocks = [
+    {
+      request: {
+        query: GET_ACCOUNT_BY_EMAIL,
+        variables: {
+          email: testEmail,
+          autoCompleted: false,
+        },
+      },
+      result: () => {
+        calledAccountSearch = true;
+        return MinimalAccountResponse(testEmail);
+      },
+    },
+  ];
+
+  const renderResult = render(
+    <MockedProvider mocks={mocks} addTypename={false}>
+      <AccountSearch />
+    </MockedProvider>
+  );
+
+  await act(async () => {
+    fireEvent.change(renderResult.getByTestId('email-input'), {
+      target: { value: testEmail },
+    });
+    fireEvent.blur(renderResult.getByTestId('email-input'));
+  });
+  await new Promise((r) => setTimeout(r, 0));
```

Comment:
Q: What does this do?
Is it like an async version of [`process.nextTick()`](https://nodejs.org/dist/latest-v16.x/docs/api/process.html#processnexttickcallback-args)?

Regardless, not sure if we want to add some comment here explaining the promisified `setTimeout()` is doin'.

Reply:
Yeah, this is something that's in the apollo docs. I also thought it was weird, but I guess it needs to settle. I'll add a comment. Here's the [doc](https://www.apollographql.com/docs/react/v2/development-testing/testing#testing-final-state), btw.

Reply:
I actually just wrapped those timeouts in a nextTick function, should be clear enough as to the intent.